typing_tests/pine-params.ts

```diff
@@ -211,13 +157,13 @@ await (async () => {
 	});
 	aNumber = result.id;
 	aString = result.device_name;
-	aNumber = result.belongs_to__application.__id;
+	aNumberOrUndefined = result.belongs_to__application?.__id;
```

Comment:
Can we do the device type thing like in the other file?
```suggestion
	aNumber = result.is_of__device_type.__id;
```